gitops/fleet/charts/argo-cd/values.yaml

```diff
@@ -0,0 +1,3 @@
+server:
+  service:
+    type: LoadBalancer
```

Comment:
Did you tested the terraform spoke destroy.sh deletes the loadbalancer here? https://github.com/aws-samples/fleet-management-on-amazon-eks-workshop/blob/riv24/terraform/spokes/destroy.sh#L29

Reply:
I found the ns for argocd on spoke incorrect in destroy.sh; I corrected that in the last commit.

Reply:
I tested the destroy.sh script after fixing the ns, and it worked.